content/en/user-manual/user-interface/localization.md

```diff
@@ -68,6 +68,23 @@ Enable the `Localized` checkbox for a Text Element in order to use the Localizat
 
 To test your localization you can change the `Locale` field under the Editor Settings. That should update your Editor viewport to that locale and also this will update the locale used when you launch your application. This field is not used when you publish or download a build.
 
+## Localizing Numbers
+
+Different locales will have different rules on how numbers should be formatted. For example, English (UK and US) would format `1000000` as `1,000,000` and Dutch would format with a decimal instead `1.000.000`.
+
+JavaScript provides a built in function to do this formatting based on the locale code, [`Number.protoype.toLocaleString()`][5].
+
+An example of usage:
+
+```javascript
+var numberOfItems = 1000;
+var currentLocale = pc.Application.getApplication().i18n.locale;
```

Comment:
```suggestion
var currentLocale = this.app.i18n.locale;
```

Reply:
I thought about this and wasn't sure if `this.app` would be 'right' as I wasn't showing the context of a script type and favoured something that could be copied and pasted anywhere

Reply:
But then 95% of developers use the Editor and should definitely be using `this.app` and never use `pc.Application.getApplication` which is horribly named (IMHO - we should alias it) and overly verbose. It might encourage devs to put that everywhere where they need the app.